### PR TITLE
[ MB-11798 ] add shipment id to query

### PR DESCRIPTION
**Pull request by kennethchow-truss**
## Summary

This PR adds the shipment id to the query as a `context_id` field. Instead of using a specific field such as shipment_id, context_id may be more helpful for when we have additional joins for `orders`, `payment_requests`, etc. 

## Setup to Run Your Code

1. Go to officelocal:3000/
2. Login as local user
3. Click on a move
4. Go to move history tab
5. Observe that the new context_id field is there

### CODE REVIEW

File: pkg/services/move_history/move_history_fetcher.go [hunk 1]
```diff
@@ -28,15 +28,6 @@ func (f moveHistoryFetcher) FetchMoveHistory(appCtx appcontext.AppContext, locat
...
 	),
-	changed_addresses AS (
-		SELECT DISTINCT
-			unnest(ARRAY [old_data->>'pickup_address_id',
-										   old_data->>'destination_address_id']) AS ID
-		FROM
-			audit_history
-			JOIN moves ON audit_history.table_name = 'mto_shipments'
-				AND audit_history.old_data ->> 'move_id' = moves.id::text
-	),
```
Comment (YanZ777) [inline pkg/services/move_history/move_history_fetcher.go:39]: Why was this deleted in this PR?

Comment (rogeruiz) [reply pkg/services/move_history/move_history_fetcher.go:39]: @kennethchow-truss could you answer @YanZ777's question above?

Comment (kennethchow-truss) [reply pkg/services/move_history/move_history_fetcher.go:39]: Changed addresses wasn't being referenced in the query, so it was unused

**Comment by kennethchow-truss** [comment]
@mergifyio rebase

**Review by YanZ777 — APPROVED** [review]
